point-of-sale/src/client/hooks/usePayment.ts

```diff
@@ -16,7 +16,6 @@ export interface PaymentContextState {
     amount: BigNumber | undefined;
     setAmount(amount: BigNumber | undefined): void;
     message: string | undefined;
```

Comment:
Do we need this as part of the Payment context since it's already on the Config context?

Reply:
Oops got a bit mixed up there. I've removed it from the payment context, it's now just consumed by `PaymentProvider` to make the URL